Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Header.css';
+import HeaderTitle from './HeaderTitle';
+
+const HeaderContent = () => {
+  return (
+    <div className="headerTextWrapper">
+      <HeaderTitle />
+      <p className="headerParagraph">Let's learn React by building simple Interfaces with components. Don't try to overthink it, just keep it simple and have fun. Once you feel comfortable using components you are well on your way to mastering React!</p>
```

Comment:
Then after passing the headerParagraph property in the HeaderContainer like I showed in the comment. You would use it here on line 9 like `<p>{props.headerParagraph}</p>`